### DISCUSSION TITLE
吐槽一下(◎_◎;)

### DISCUSSION THREAD

**Discussion by 2007lxyx** [NapNeko/NapCatQQ#601] [Q&A] [answered]
![F82C6B9D921600C953EB0C980F4A581A](https://github.com/user-attachments/assets/e3535aad-f241-4025-9759-50cf81f42f65)
nt 文件类型仅支持图片，发送可以识别不行，大佬能不能解决一下？

**Comment by MliKiowa** [comment, marked as answer]
> ![F82C6B9D921600C953EB0C980F4A581A](https://private-user-images.githubusercontent.com/177051015/392446355-e3535aad-f241-4025-9759-50cf81f42f65.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzMjU2NjYsIm5iZiI6MTczMzMyNTM2NiwicGF0aCI6Ii8xNzcwNTEwMTUvMzkyNDQ2MzU1LWUzNTM1YWFkLWYyNDEtNDAyNS05NzU5LTUwY2Y4MWY0MmY2NS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNFQxNTE2MDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNmEyODQ1ODY2NDI0NGI5ZjY0MzEzYzcyYmQyY2ViMzIzZWI0YThkMjljMGQ0Mzc1ZjI3YjI4YjVjNTMzMWViJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.OJYVBxUNYHdRVpqAgdUYeutxlB2Tvwri9yZPoZtMzQU) nt 文件类型仅支持图片，发送可以识别不行，大佬能不能解决一下？

可以拿到 接口大概是下面这样 我记不清了 通过gocq get_group_file_url的接口

**Comment by 2007lxyx** [comment]
谢谢大佬( ˘ ³˘)❤



---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 日期 | 2024年12月04日 23:17 |
| 收件人 | ***@***.***> |
| 抄送至 | ***@***.***>***@***.***> |
| 主题 | Re: [NapNeko/NapCatQQ] 吐槽一下(◎_◎;) (Discussion #601) |

nt 文件类型仅支持图片，发送可以识别不行，大佬能不能解决一下？

可以拿到 通过gocq get_group_file_url的接口
